lua/neotest/consumers/output.lua

```diff
@@ -5,9 +5,15 @@ local config = require("neotest.config")
 local win, short_opened
 
 local function open_output(result, opts)
+  local output = opts.short and result.short or (result.output and lib.files.read(result.output))
+  if not output then
+    if result.output ~= "<stream>" then
```

Comment:
@rcarriga - not sure if this is acceptable, but I've added a check for whether `open_output` is being called on a stream result where we don't expect there to be a filename.

Reply:
Not a fan of this, it means that all consumers need to be aware that `<stream>` has a special meaning. Instead we can add a new option:
```lua
    if not opts.quiet then
      lib.notify("Output not found for position", "warn")
    end
```
and then below in the `results` listener
```lua
            { enter = false, short = config.output.open_on_run == "short", quiet = true }
```
Also add the option to the docs 
```lua
---@field quiet boolean: Suppress warnings of no output
```

Reply:
no worries - done :)